include/m_private.h

```diff
@@ -24,49 +26,113 @@ typedef enum mPr_ITEM_COND{
     /* 3 */ mPr_ITEM_COND_NUM
 } mPr_ITEM_COND;
 
-#define mPr_POCKETS_SLOT_COUNT 15
-#define mPr_DELIVERY_QUEST_NUM mPr_POCKETS_SLOT_COUNT
-#define mPr_ERRAND_QUEST_NUM 5
-
-typedef struct Private_Sub_A86 {
-    /* 0x00 */ char unk00[0x3];
-    /* 0x03 */ u8 unk_03;
-    /* 0x04 */ char unk04[0x1];
-    /* 0x05 */ u8 unk_05;
-    /* 0x06 */ u16 unk_06;
-    /* 0x08 */ u8 unk_08;
-} Private_Sub_A86; // size >= 0xA
-
-typedef struct mPr_map_info_c {
-  /* 0x00 */ char unk00[0x8];
-} mPr_map_info_c; // size = 0x8
-
-typedef struct PrivateInventory {
-    /* 0x00 */ u16 pockets[mPr_POCKETS_SLOT_COUNT]; /* items in inventory */
-    /* 0x1E */ u8 lotto_ticket_expiry_month;
-    /* 0x1F */ u8 lotto_ticket_mail_storage;
-    /* 0x20 */ u32 item_conditions; /* bitfield (15 values), 2 bits per pocket slot */
-    /* 0x24 */ u32 wallet;
-    /* 0x28 */ u32 loan;
-} PrivateInventory; // size = 0x2C
-
 typedef struct PrivateInfo {
-    /* 0x000 */ UNK_TYPE1 unk000[0x10];
+    /* 0x000 */ PersonalID_c playerId; 
     /* 0x010 */ s8 gender;
     /* 0x011 */ UNK_TYPE1 unk011[0x3];
     /* 0x014 */ PrivateInventory inventory;
     /* 0x040 */ mQst_delivery_c deliveries[mPr_DELIVERY_QUEST_NUM]; /* delivery quests */
-    /* 0x25C */ mQst_errand_c errands[mPr_ERRAND_QUEST_NUM]; /* errand quests */
+    /* 0x25C */ QuestErrand errands[mPr_ERRAND_QUEST_NUM]; /* errand quests */
     /* 0x3EC */ UNK_TYPE1 unk_3EC[0x2];
     /* 0x3EE */ MailHeaderCommon unk_3EE;
-    /* 0x40A */ Mail unk_40A[10];
+    /* 0x40A */ Mail_c unk_40A[10];
     /* 0xA72 */ UNK_TYPE1 unk_A72[0x2];
     /* 0xA74 */ u8 exists;
     /* 0xA75 */ UNK_TYPE1 unkA75[0x11];
     /* 0xA86 */ Private_Sub_A86 unk_A86;
-    /* 0xA90 */ UNK_TYPE1 unkA8F[0xF8];
+    /* 0xA8F */ s8 unkA8F[0x3c];
```

Comment:
is this an array of `s8`? or is it a placeholder? If it is the latter, can you use `UNK_TYPE1` instead?